docs/isoSPI.md

```diff
@@ -1,3 +1,109 @@
+
+<img src="images/amdc-isoSPI.svg" />
+
+### 1. DB15 Connector
+
+DB15 D-sub high-density connectors are used for isoSPI and differential I/O (D_I / D_O) interface in AMDC. There are 2 DB15 connectors and each connector has 2 isoSPI communication interface, 2 differential I/O interface, 1 pin for ground signals, and 1 pin for 5V supply from the AMDC.
+
+The pin mappings for each DB15 connector is shown:
+
+| Pin number | Signal name |
+|------------|--------|
+| 1 | 5V |
+| 2 | D1_I_P |
+| 3 | D1_I_N |
+| 4 | D2_I_P |
+| 5 | D2_I_N |
+| 6 | NC |
+| 7 | isoSPI1_P |
+| 8 | isoSPI1_N |
+| 9 | isoSPI2_P |
+| 10 | isoSPI2_N |
+| 11 | GND |
+| 12 | D1_O_P |
+| 13 | D1_O_N |
+| 14 | D2_O_P |
+| 15 | D2_O_N |
+
+The mapping between the AMDC schematic labels, PicoZed pins, and Zynq-7000 balls used in Vivado, can be found at the isoSPI section of the [pin mapping document](RevD-PinMapping.md#encoder).
+
+Location of these connectors in the AMDC is shown below:
+
+<img src="images/amdc-isoSPI-input-highlighted.svg" />
+
+More details on the DB15 connector can be found in [here](https://content.norcomp.net/rohspdfs/Connectors/17Y/178/513/178-H15-513R497.pdf)
+
+### 2. isoSPI communication interface
+
+The isoSPI communication interface is implemented using [LTC6820](https://www.analog.com/media/en/technical-documentation/data-sheets/LTC6820.pdf). This IC provides a bi-directional interface between standard SPI signals and differential pulses. The SPI signals from PicoZed are in 1.8V voltage level, this IC also translates the signals to 5V level. For termination, 120 ohms resistor is added. The maximum and minimum operating conditions are provided in the following table:
+
+| Parameter                             |    Conditions     |   MIN   |  MAX  |
+|---------------------------------------|-------------------|---------|-------|
+| Supply voltage VDD (Differential side)|                   | 2.7 V   | 5.5 V |
+| IO supply voltage VDDS (SPI side)     |                   | 1.7 V   | 5.5 V |
+| High-level input voltage              |VDDS = 2.7V to 5V  | 0.7 VDDS| VDDS  |
+| High-level input voltage              |VDDS = 1.7V to 2.7V| 0.8 VDDS| VDDS  |
+| Low-level input voltage               |VDDS = 2.7V to 5V  | 0       | 0.3 VDDS  |
+| Low-level input voltage               |VDDS = 1.7V to 2.7V| 0       | 0.2 VDDS  |
+
+This IC can operate at a maximum SPI communication speed of 1 Mbps. The bias resistors (RB1 and RB2) are used to adjust the drive currents to the differential lines. The bias resistors (RB1 and RB2) used in the design will set the drive currents to 10 mA. The maximum supply current consumed by the IC including to drive currents for differential lines is 15.8 mA which corresponds to 79 mW for 5 V supply. More information regarding the operating conditions, bias resistors, maximum throughput rate, can be found in the [datasheet](https://www.analog.com/media/en/technical-documentation/data-sheets/LTC6820.pdf). 

```

Comment:
> The bias resistors (RB1 and RB2)

Provide a simple diagram showing where `RB1` and `RB2` appear.

Reply:
Added the bias resistors block diagram